pkg/cidata/cidata.go

```diff
@@ -126,6 +126,18 @@ func GenerateISO9660(instDir, name string, y *limayaml.LimaYAML, udpDNSLocalPort
 			return err
 		}
 		args.Mounts = append(args.Mounts, expanded)
+		option := "ro"
+		if *f.Writable {
+			option = "rw"
+		}
+		args.MountOptions = append(args.MountOptions, option)
+	}
+
+	switch *y.MountType {
+	case limayaml.REVSSHFS:
+		args.MountType = "sshfs"
```

Comment:
```suggestion
		args.MountType = "reverse-sshfs"
```

Reply:
There is no mount type with that name, `mount -t reverse-sshfs`

Reply:
That is why I suggested renaming the user-facing option, to something else.

Reply:
"sshfs" is reserved for real sshfs, i.e., the guest works as a SSH client, and the host works as a SSH server.

Reply:
Renamed it to "reverse-sshfs", it is not really running any mount commands anyway